add kubernetes deploy example

TheKangaroo opened the pull request:
I deployed the heartbeat proxy to kubernetes yesterday and wanted to share the examples with you.

tilladamdmde committed:
add kubernetes deploy example

jayme-github commented on examples/k8sdeploy.yaml:
```diff
@@ -0,0 +1,45 @@
+    spec:
+      containers:
+        - name: heartbeat-proxy
+          image: registry.gitlab.dm-drogeriemarkt.com/docker/opsgenie-heartbeat-proxy
```
Could you please exchange the `image` for the autobuild one from https://hub.docker.com/r/traumfewo/opsgenie-heartbeat-proxy/

jayme-github commented on examples/k8sdeploy.yaml:
```diff
@@ -0,0 +1,45 @@
+  labels:
+    app: heartbeat-proxy
+  name: heartbeat-proxy
+  namespace: monitoring
```
Please remove the `namespace:` to make this a little more generic.

jayme-github reviewed (changes requested):
Hi,
thank you very much for your contribution!

I will forward this to @NilsGriebner for further review and merge. Do you mind adding a small note to the README regarding your example?

NilsGriebner commented:
@TheKangaroo thanks for your contribution :) I agree with @jayme-github points. The changes would be great. We use resource limits in our deployment, maybe we should add it here too. What do you think?

NilsGriebner commented on examples/k8sdeploy.yaml:
```diff
@@ -0,0 +1,45 @@
+            value: "youropsgenieapikey"
+          - name: HEARTBEAT_NAME
+            value: "yourheartbeatname"
+#          - name: DEBUG
```
I think this could be removed, because it's for debugging only.

tilladamdmde committed:
add kubernetes docs

TheKangaroo commented:
Hey @NilsGriebner, @jayme-github , sorry for the delay. I fixed the image path and removed the namespace.
I also added the limits. We use them too in our setup, but I'm not 100% sure if we really should add them to a simple example. I'm fine with both ways, you guys decide ;-)

I added some lines about kubernetes to the readme too.
What do you think?

NilsGriebner commented:
Hi @TheKangaroo looks good to me now :) I will give it a try in our playground and merge if I don't spot any issues.

Thanks aggain, great work :)
